subctl/diagnore-pre-install.md

```diff
@@ -0,0 +1,113 @@
+# Subctl `diagnose pre-install`
+
+## Summary
+
+As a user of Submariner, it would be useful in a myriad of situations to have a tool capable of testing the Kubernetes infrastructure
+where Submariner is to be installed.
+The tool should check all of Submariner's pre-requisites from the infrastructure, and report any deficiencies.
+It would be nice if the tool can also offer guidance for the deficiencies, so that the user can address them and install Submariner successfully.
+
+This tool would compliment other `subctl diagnose` commands which check that Submariner is installed and working properly, by allowing
+similar checks **before** the actual installation.
+
+Alternatively, the new command could be hosted elsewhere, if nesting under `diagnose` is deemed confusing.
```

Comment:
I think it should be elsewhere, ie "diagnose" connotes troubleshooting after installation.